connect/src/main/java/com/stripe/android/connect/StripeComponentController.kt

```diff
@@ -1,32 +1,94 @@
 package com.stripe.android.connect
 
 import androidx.annotation.RestrictTo
+import androidx.annotation.VisibleForTesting
+import androidx.fragment.app.FragmentActivity
+import androidx.lifecycle.ViewModelProvider
 
 /**
  * Controller for a full screen component.
  */
 @PrivateBetaConnectSDK
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-interface StripeComponentController<Listener : StripeEmbeddedComponentListener> {
+abstract class StripeComponentController<Listener, Props> internal constructor(
```

Comment:
👀 As mentioned earlier, this was changed to an abstract class with an internal constructor. Its implementation was merged in from `StripeComponentControllerImpl`, which was deleted. There are no logic changes.